lib/cmds/space_cmds/use.js

```diff
@@ -25,7 +25,7 @@ export const builder = (yargs) => {
 }
 
 function showSuccess (space) {
-  success(`Now using Space ${highlightStyle(space.name)} (${highlightStyle(space.sys.id)}) when the \`--space-id\` option is missing.`)
```

Comment:
Hey @floelhoeffel Actually we need that, to let the user know that they override the default space id